### DISCUSSION TITLE
[HW3] About Prob1.2

### DISCUSSION THREAD

**Discussion by emjay73** [seominjoon/kaist-ai605#138] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/36818801/140864694-2e98aa3f-6815-46cd-b379-72233883bb0c.png)

Q and K is 1 x dk matrix?
Otherwise, 1 in the following equation means identity matrix?
![image](https://user-images.githubusercontent.com/36818801/140864626-27debf40-3383-43ad-9e07-b3caf69df896.png)

**Comment by seominjoon** [comment, marked as answer]
Here "1" is a matrix of 1's with the same size as the left side.

**Reply by emjay73** [reply]
In case when Q is axb and K is cxb then 1 means axc dimension matrix full of ones??

**Reply by seominjoon** [reply]
Yes, you are correct!

**Reply by emjay73** [reply]
Thank you, professor!
